.github/workflows/release-publish.yml

```
@@ -19,7 +19,7 @@ jobs:
         fetch-depth: 0
     - uses: fregante/setup-git-user@v1
 
-    - uses: actions/setup-node@v1
+    - uses: actions/setup-node@v2.4.0
```

```suggestion
    - uses: actions/setup-node@v2
```
Maybe you'll prefer the major version only